equinix/resource_network_device.go

```diff
@@ -371,6 +431,7 @@ func createNetworkDeviceSchema() map[string]*schema.Schema {
 		networkDeviceSchemaNames["VendorConfiguration"]: {
 			Type:     schema.TypeMap,
 			Optional: true,
+			Computed: true,
```

Comment:
This was added to accommodate additional vendor configuration details that are included in the API response.